pyaedt/hfss.py

```diff
@@ -2001,12 +2041,12 @@ def edit_source(self, portandmode, powerin, phase="0deg"):
 
     @aedt_exception_handler
     def thicken_port_sheets(self, inputlist, value, internalExtr=True, internalvalue=1):
-        """Create thickened sheets over a list of input port faces.
+        """Create thickened sheets over a list of input port sheets.
```

Comment:
@maxcapodi78 I did the modification regarding the description as we previously discussed.

Reply:
@MaxJPRey Great. Thank you